src/core/css/SearchResult.scss

```diff
@@ -48,19 +48,13 @@
   }
 }
 
-.SearchResult-author,
-.SearchResult-users {
+.SearchResult-author-users {
   color: $sub-text-color;
-  display: inline-block;
+  display: block;
   font-size: $font-size-s;
   font-weight: normal;
-  margin: 0;
+  margin-right: 0.5rem;
```

Comment:
Why are `rem` used here but `em` elsewhere? And in general I've liked our `px` values, they are easy to understand and follow the mocks well.